### PR TITLE
fixing issue with participant requests

### PR DESCRIPTION
**Pull request by chelpu**

### CODE REVIEW

**Comment by natebrennand** [comment]
can you make sure to run `go fmt ./...`?
lgtm to me otherwise, if you wanted to add a callback handler for conference callbacks it could be useful
https://www.twilio.com/docs/api/twiml/conference#attributes-eventCallbackUrl

File: conference/participant.go [hunk 1]
```diff
@@ -49,8 +49,8 @@ func (act Account) Participant(confSid string, callSid string) (Participant, err
 	} else if !validateCallSid(callSid) {
 		return p, errors.New("Invalid call sid")
 	}
-
-	err := common.SendGetRequest(fmt.Sprintf(getURL, act.AccountSid, confSid, callSid), act, &p)
+	err := common.SendGetRequest(fmt.Sprintf(participantURL, act.AccountSid, confSid, callSid), act, &p)
+	fmt.Println("p: ", p.CallSid)
```
Comment (natebrennand) [inline conference/participant.go]: can we scrap this too?

Comment (chelpu) [inline conference/participant.go]: yup

**Comment by chelpu** [comment]
Will try to finish up the callback tomorrow as well as the rest of incoming phone numbers

**Comment by natebrennand** [comment]
lgtm unless you wanna update the callbackHandler to be non-blocking like here: https://github.com/natebrennand/twiliogo/pull/54/files

**Pull request merged**